runtimes-common/attributes/runtimes-attributes.adoc

```diff
@@ -10,13 +10,17 @@
 //
 
 // Each Runtimes product needs a unique name attribute for shared content.
+// When you write shared content, use the following attributes instead of the regular product name attributes of your project to ensure that correct product names are substituted when you reuse content across different documentation sets. 
 
 :rhdg-long: Red Hat Data Grid
 :rhdg-short: Data Grid
 
 :jws-long: Red Hat JBoss Web Server
 :jws-short: JBoss Web Server
 
+:runtimes-rhbq-long: Red Hat build of Quarkus
+:runtimes-rhbq-short: Quarkus
```

Comment:
@inoxx03 Yes, let's add the "runtimes-" prefix to these attributes to make them easier to distinguish with shared content.

I think it would also be easier for other writers if we avoided 4 letter abbreviations that are identifiable by project writers. An easy mistake might be typing "rhbq" instead of "rhdg". 

What do you think about ":runtimes-quarkus-long:" instead? Then maybe ":runtimes-datagrid-long:" etc.

Reply:
@oraNod Good point! Product acronyms are convenient, but might be unfamiliar to some writers.
Also, since we reuse the content across downstream docs only, there is IMHO little potential for confusion about whether the name refers to upstream or downstream.

I'll change the acronyms to more "friendly" names:)